apitools/base/py/transfer.py

```diff
@@ -541,6 +544,25 @@ def StreamMedia(self, callback=None, finish_callback=None,
         self._ExecuteCallback(finish_callback, response)
 
 
+if six.PY3:
+    class MultipartBytesGenerator(email_generator.BytesGenerator):
+        """Generates a bytes version of a Message object tree for multipart messages
```

Comment:
Linter will probably complain that this line is more than 80 chars long.  Also needs an extra line of whitespace between the summary line and the description paragraph.

Reply:
Good catch, fixed.